src/main/resources/ccw-database-data.sql

```diff
@@ -6,23 +6,23 @@ VALUES
 ;
 
 ------------------ family law -----------------------
-INSERT INTO CCW.MATTER_CODES_1(MATTER_CODE_ID, LAW_TYPE)
+INSERT INTO CCW.MATTER_CODES_1(MATTER_CODE_ID, LAW_TYPE, DESCRIPTION)
```

Comment:
As noticed by @smallmanj-moj, now that the description column is populated, `@Mapping(target = "description", defaultValue = "")` needs to removed from `MatterCodeMapper`.